openfisca_core/parameters.py

```diff
@@ -509,6 +508,20 @@ def add_child(self, name, child):
         self.children[name] = child
         setattr(self, name, child)
 
+    def __repr__(self):
+        return os.linesep.join(
+            [os.linesep.join(
+                ["{}:", "{}"]).format(name, indent(repr(value))).encode('utf-8')
+                for name, value in self.children.iteritems()]
+            )
+
+
+class Bracket(ParameterNode):
```

Comment:
Can we put this in the  `Scale` class ?

Reply:
Nop, sometimes users want to add a new Bracket to a scale, and they need to access the class.